README.quickstart.vscode.md

```diff
@@ -92,4 +104,214 @@ For a full list of features, visit [our documentation](https://docs.aws.amazon.c
 
 ## Troubleshooting
 
-Submit bug reports and feature requests [on our Github repository](https://github.com/aws/aws-toolkit-vscode/issues/new/choose).
+![Configure and Run](./resources/marketplace/vscode/sam-configure-and-run-still.png)
+
+###### After a local run is complete, the output appears in the **OUTPUT** tab.
+
+When you're satisfied with performance, you can [deploy your serverless application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app). The SAM template is converted to a CloudFormation template, which is then used to deploy all the application's assets to the AWS Cloud.
+
+### Launch config auto-completion ("IntelliSense")
+
+The `Add Debug Configuration` _CodeLens_ creates launch configs of type
+`aws-sam` in the VS Code `launch.json` file. You can also create these entries
+by hand.
+
+When editing `launch.json` configs, AWS Toolkit provides auto-completion and
+contextual documentation, as shown below.
+
+<img src="./resources/marketplace/vscode/sam-launch-config.png" alt="Debug Configuration inline documentation" width="640"/>
+
+### Supported runtimes
+
+The Toolkit _local SAM debugging_ feature supports these Lambda runtimes:
+
+-   C# (.NET Core 2.1, 3.1; .NET 5.0, .NET 6.0)
+-   Go (1.x)
+-   Java (8, 8.al2, 11)
+-   JavaScript/TypeScript (Node.js 12.x, 14.x)
+-   Python (3.6, 3.7, 3.8, 3.9, 3.10)
```

Comment:
We no longer list this info in this file (all changes to this file can be dropped)